src/Components/Movies/Movies.js

```diff
@@ -2,25 +2,27 @@ import React from 'react'
 import './Movies.scss'
 import Movie from '../Movie/Movie'
 
-const Movies = ({movies, handleClick}) => {
-  const movieCards = movies.movies.map(movie => {
-    return (
-      <Movie 
-        image={movie.poster_path}
-        // title={movie.title}
-        // tagline={movie.tagline}
-        id={movie.id}
-        key={movie.id}
-        handleClick={handleClick}
-      />
+const Movies = ({movies, getSingleMovieData, isLoading}) => {
+  if (isLoading) {
```

Comment:
Re: our other question on {movies: []}, here is where we could use a conditional testing for if the array is truthy/falsey rather than isLoading.